### PR TITLE
Rspec tictac

### PR DESCRIPTION
**Pull request by arikarim**
Added Rspect tests to methods in main, board and display,.

### CODE REVIEW

File: spec/test.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
+require_relative '../lib/board'
+require_relative '../bin/main'
+require_relative '../bin/Rspec.file'
+# require_relative '../lib/display'
+# require_relative '../lib/game'
```
Comment (rukundoeric) [inline spec/test.rb]: remove this one too

File: spec/test.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
+require_relative '../lib/board'
+require_relative '../bin/main'
+require_relative '../bin/Rspec.file'
```
Comment (rukundoeric) [inline spec/test.rb]: I checked `bin/` directory, but I couldn't find `Rspec.file` in there. if it is a mistake, kindly remove it.

File: bin/rspec_file.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# def user_input(current_player)
+#   board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+#   board = Board.new
+#   if board.valid_move?(2)
+#     board.update_board(2, 'X')
+#   else
+#     if_invalid('denis')
+#     user_input(current_player)
+#   end
+# end
```
Comment (rukundoeric) [inline bin/rspec_file.rb]: Please remove this commented code.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are some changes required, please check the comments below._

- ### To Highlight
  - [x] The project is clean 💯
  - [x] It has a descriptive branch name 💯
  - [x] The project follows Git flow. 💯
- ### To Improve
  - [x] [MANDATORY] Please remove all commented codes.
  - [x] [MANDATORY] when I run `rspec or rspec --format d` this is the out put I'm getting 
Capture please make sure your tests are executable not only in your local computer.

_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on @slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
@slack: Rukundo Eric**

**Comment by arikarim** [comment]
thanks for the review we did all the changes
please have a look again

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - [x] The project is clean. 💯
  - [x] The project follows Git flow. 💯
  - [x] It has a descriptive branch name 💯
  - [x] The Game logic works great. 💯
  - [x] All tests are passing. 💯
- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge to master and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), @slack: Rukundo Eric**

**Pull request merged**